pkg/ivd/ivd_protected_entity_type_manager.go

```diff
@@ -40,36 +41,47 @@ type IVDProtectedEntityTypeManager struct {
 	vsom      *vslm.GlobalObjectManager
 	cnsClient *cns.Client
 	s3URLBase string
-	user      string // These are being kept so we can open VDDK connections, may be able to open a VDDK connection
-	password  string // in IVDProtectedEntityTypeManager instead
+	vcParams  map[string]interface{} // Save the VC configuration params
```

Comment:
Apart from user password what else are we expecting here?. Is map[string]interface{} necessary? or should map[string]string suffice?

Reply:
I think it makes sense, but how about we make this change as a different commit? I am wondering why we use map[string]interface{} in the first place in the plugin code.

Reply:
Some of the other Protected Entities have more complex parameters and can pass down things like arrays or maps as single keys.